templates/financial-services/resources.html

```diff
@@ -27,239 +31,309 @@ <h2>Canonical &ndash; Trusted source of open source for financial services</h2>
       </div>
-      <div class="col-7">
-        <p>Financial institutions are increasingly pressed for agility and velocity to adapt to changing market conditions, increased customer expectations while satisfying regulatory and compliance requirements.</p>
-        <p>Canonical helps financial institutions drive infrastructure efficiency and operational agility with multi-cloud, hybrid cloud and bare-metal solutions.</p>
+      <div class="col-3 u-hide--small u-hide--medium">
+        {{ image (
+          url="https://assets.ubuntu.com/v1/05dff9cb-WH5.png",
+          alt="",
+          width="1700",
+          height="2200",
+          hi_def=True,
+          loading="lazy"
+          ) | safe
+        }}
       </div>
     </div>
-  </article>  
+  </article>
 </section>
 
-<section class="p-strip u-no-padding--bottom">
+<section class="p-strip">
   <div class="u-fixed-width">
     <h2>Blogs</h2>
-    <p>All Canonical blogs related to open source in Financial Services are <a href="/blog/tag/financial-services">here</a>. This rich set of content covers everything from high-level product announcements to hands-on technical guides that dig into more complicated topics.</p>
+    <p class="u-sv2">All Canonical blogs related to open source in Financial Services are <a href="/blog/tag/financial-services">here</a>. This rich set of content covers everything from high-level product announcements to hands-on technical guides that dig into more complicated topics.</p>
   </div>
-</section>
 
-    {% with section_classes='p-strip', heading_topic='Financial Services', tag_name='	financial-services', tag_id='3955 ', limit='4' %}
-      {% include "shared/_latest_news_strip.html" %}
-    {% endwith %}
+  <div id="latest-articles" class="row">
+    <div style="min-height: 9.1rem">
+      <i class="p-icon--spinner u-animation--spin">Loading...</i>
+    </div>
+  </div>
+
+  <template style="display:none" id="articles-template">
+    <article class="col-4 col-medium-2 blog-p-card--post">
+      <div class="blog-p-card__content">
+        <div class="u-crop--16-9">
+          <div class="article-image" aria-hidden="true"></div>
+        </div>
+        <h3 class="p-heading--4">
+          <a class="p-link--soft article-title article-link"></a>
+        </h3>
+        <p><em class="article-time"></em></p>
+      </div>
+    </article>
+  </template>
+
+  <script src="{{ versioned_static('js/dist/latest-news.js') }}"></script>
+  <script>
+    canonicalLatestNews.fetchLatestNews(
+      {
+        articlesContainerSelector: "#latest-articles",
+        articleTemplateSelector: "#articles-template",
+        limit: "3",
+        gtmEventLabel: "financial-services",
+        tagId: 3955
+      }
+    )
+  </script>
+</section>
 
 <section class="p-strip--light">
-  <div class="u-fixed-width">
-    <h2>Podcasts</h2>
-  </div>
   <div class="row">
-    <div class="col-5">
+    <div class="col-8">
+      <h2>Podcasts</h2>
```

Comment:
The copy doc says "Podcast" in singular, I suppose it's correct, since there is only 1